spike_tools/general.py

```diff
@@ -52,32 +64,46 @@ def get_channels_list(fs, subject, session, region="all"):
         pfc_channels = np.array([x for x in all_channels if x.find('a') >= 0])
         return pfc_channels
     
-def list_session_units(fs, subject, session):
-    spike_path = get_spike_path(subject, session)
-    unit_files = fs.glob(spike_path + "*spiketimes.mat")
+def list_session_units(fs, subject, session, species_dir=NHP_WCST_DIR):
+    spike_path = get_spike_path(subject, session, species_dir)
+    if fs is None: 
+        unit_files = glob.glob(spike_path + "*spiketimes.mat")
```

Comment:
Minor note, glob returns files in an arbitrary order. Maybe sort them after globbing?